app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,14 @@
         }
       });
     </script>
+
+  <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-TNHHHPZ');</script>
```

Comment:
If this was moved to a separate file, I'd try to pull out this `GTM-TNHHHPZ` identifier a bit - assuming it's the tracking code for "our" google analytics. Will hopefully make it easier to understand / change in the future.

e.g. (pseudocode)
```javascript
var trackingCode = 'GTM-TNHHHPZ'
...
(window,document,'script','dataLayer',trackingCode)
```

This is a minor one in comparison to the file separate, though.